test/module/s3.spec.js

```diff
@@ -209,27 +211,21 @@ describe('Testing s3 Util', {
     expect(result).to.equal('2018-10-25T20:55:00.000Z/Collection Viewed.json.gz');
   });
 
-  it('Testing error rate exceeds retry count', async ({ capture, recorder }) => {
+  it('Testing error rate does not exceed retry count', async ({ recorder }) => {
     const s3 = S3({
       logger: console,
       backoffFunction: () => 0,
-      maxRetries: 0
+      maxRetries: 1
     });
-    const error = await capture(() => s3.putGzipObject({
+    const result = await s3.putGzipObject({
       bucket,
       key,
       data: JSON.stringify({ data: 'data' })
-    }));
-    expect(error).to.deep.contain({
-      statusCode: 503
     });
-    expect(recorder.get()).to.deep.equal([
-      'Failed to submit s3:putObject\n'
-      + '{"errorCode":"SlowDown","action":"s3:putObject","opts":{"ContentType":"application/json",'
-      + '"ContentEncoding":"gzip","Bucket":"test-bucket-name","Key":"key","Body":{"type":"Buffer",'
-      + '"data":[31,139,8,0,0,0,0,0,2,3,171,86,74,73,44,73,84,178,130,80,181,0,185,30,67,221,15,0,0,0]}},'
-      + '"retryCount":0}'
-    ]);
+    expect(result).to.deep.equal({});
+    const errorLog = recorder.get();
+    const errorLogSplit = errorLog.map((e) => e.split('\n')[0]);
+    expect(errorLogSplit).to.deep.equal(['Request failed for s3.putObject()']);
```

Comment:
These logs are quite long. Is there a way you would want to see them?

Reply:
I'd like to see the retry count. How long are we talking? I'd just have them in here